web/app/js/components/ResourceDetail.jsx

```diff
@@ -14,15 +15,19 @@ import TopRoutesTabs from './TopRoutesTabs.jsx';
 import Typography from '@material-ui/core/Typography';
 import _filter from 'lodash/filter';
 import _get from 'lodash/get';
+import _indexOf from 'lodash/indexOf';
 import _isEmpty from 'lodash/isEmpty';
 import _isEqual from 'lodash/isEqual';
 import _isNil from 'lodash/isNil';
 import _merge from 'lodash/merge';
 import _reduce from 'lodash/reduce';
+import { processEdges } from './util/EdgesUtils.jsx';
 import { withContext } from './util/AppContext.jsx';
 
 // if there has been no traffic for some time, show a warning
 const showNoTrafficMsgDelayMs = 6000;
+// resource types supported when querying API for edge data
+const possibleResourceType = ["daemonset", "deployment", "job", "pod", "replicationcontroller", "statefulset"];
```

Comment:
nit: can we call this "edgeDataAvailableType" or `edgeDataAvailable` or something? `possible` seems kind of vague